### PR TITLE
executor: migrate test-infra to testify for chunk_size_control

### LINKED ISSUE #28569: migrate test-infra to testify for `chunk_size_control_test.go`

### PR DESCRIPTION
**Pull request by hawkingrei**
Signed-off-by: Weizhen Wang <wangweizhen@pingcap.com>

### What problem does this PR solve?

Issue Number: close #28569

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: executor/chunk_size_control_test.go [hunk 1]
```diff
@@ -121,8 +118,7 @@ type testChunkSizeControlSuite struct {
 	m map[string]*testChunkSizeControlKit
 }
 
-func (s *testChunkSizeControlSuite) SetUpSuite(c *C) {
-	c.Skip("not stable because coprocessor may result in goroutine leak")
+func (s *testChunkSizeControlSuite) SetUpSuite(t *testing.T) {
```
Comment (tisonkun) [inline executor/chunk_size_control_test.go:121]: I'd prefer a `createChunkSizeControlSuite(t *testing.T) *ChunkSizeControlSuite` method.

Comment (tisonkun) [reply executor/chunk_size_control_test.go:121]: I'm pushing a patch on this now...

**Comment by tisonkun** [comment]
CI failed exactly on the changed tests.

**Comment by tisonkun** [comment]
@hawkingrei I tend to migrate the test first and file another issue to remove `t.Skip` for these tests.

**Comment by hawkingrei** [comment]
> @hawkingrei I tend to migrate the test first and file another issue to remove `t.Skip` for these tests.

OK, I have added `t.Skip`.

**Review by LittleFall — APPROVED** [review]

**Pull request merged**